atomics/T1580/T1580.yaml

```diff
@@ -54,3 +54,45 @@ atomic_tests:
       rm -rf stratus*
     name: sh
     elevation_required: false
+    
+- name: AWS - EC2 Security Group Enumeration
+  description: "Simulate an attacker's action to enumerate EC2 Security Groups in a compromised AWS environment."
+  supported_platforms:
+    - linux
+    - macos
+    - windows
```

Comment:
Hello @prashanthpulisetti  Thanks for your contribution. This atomic should belong only in `iaas:aws` platform and you should be able to execute it any of the platforms(windows, macos and linux). You can remove these platforms here

Reply:
Hi @cyberbuff , thanks for your inputs, let me update the file. Thanks!!!

Reply:
Thanks for the fix :)